docs/book/features/orphan_snapshots.md

```diff
@@ -0,0 +1,71 @@
+## Orphan snapshots detection and clean-up
```

Comment:
Can you get this documentation reviewed by @xing-yang and Deepak also?

Reply:
ok, will ask them